spec/support/page_objects/eligiblity_interface/start.rb

```diff
@@ -0,0 +1,15 @@
+module PageObjects
+  class Start < Base
+    set_url "/eligibility"
+
+    element :heading, "h1"
+
+    element :sign_in, ".app-start-page-banner__button"
```

Comment:
We should probably call this `:signin_button` or `:start_button` So it'll ready `start_page.start_button.click` in the specs.